src/lib/builders/menu/create.ts

```diff
@@ -252,7 +252,8 @@ export function createMenuBuilder(opts: _MenuBuilderOptions) {
 					 */
 					const isCharacterKey = e.key.length === 1;
 					const isModifierKey = e.ctrlKey || e.altKey || e.metaKey;
-					if (!isModifierKey && isCharacterKey && get(typeahead) === true) {
+					const isBackspaceKey = e.key === 'Backspace';
+					if (!isModifierKey && (isCharacterKey || isBackspaceKey) && get(typeahead) === true) {
```

Comment:
for consistency, should this use `kbd.BACKSPACE` constant instead? how about moving `isModifierKey`,  `isCharacterKey` and `isBackspaceKey`  to `src/lib/internal/helpers/keyboard.ts` as helper functions?

Reply:
I added all the suggested changes :)